recce/dbt.py

```diff
@@ -623,6 +624,52 @@ def pick_columns_to_compare(c1, c2):
             result = dict(
                 summary=dict(total=total, added=added, removed=removed),
                 data=json.loads(df.to_json(orient='table', index=False)),
-                raw=column_groups
+                raw=column_groups,
+                errors=errors
             )
             return result
+
+    def verify_primary_key(self, primary_key: str, model: str):
+        errors = []
+
+        def callback(check_name, executor, sql, is_base: bool):
+            table = executor(sql)
+            invalids = len(table.rows)
```

Comment:
Please don't pull all the invalids data to local. It would take long time. See how dbt test work

https://app.snowflake.com/gkfbopt/vg68680/#/compute/history/queries/01b13db4-0001-96dd-0001-40ae01fb7a1a/detail?autoRefreshInSeconds=0

It only get the count, rather than individual records.

Reply:
update the check by counting